q2_types/feature_table/_transformer.py

```diff
@@ -106,3 +106,8 @@ def _6(data: biom.Table) -> BIOMV210Format:
     with ff.open() as fh:
         data.to_hdf5(fh, generated_by=_get_generated_by())
     return ff
+
+
+@plugin.register_transformer
+def _7(data: biom.Table) -> pd.DataFrame:
+    return _table_to_dataframe(data)
```

Comment:
That's an interesting idea! I don't have much of an opinion on this, maybe it's a little safer to leave as-is in case transformer registration mutates the registered function in the future (not sure when/why we'd want to do that though).